tuxemon/event/conditions/monster_property.py

```diff
@@ -9,31 +9,36 @@
 
 class MonsterPropertyCondition(EventCondition):
     """
-    Check to see if a monster property or condition is as asked.
+    Check to see if a monster in the party has one of
+    the following property.
 
     Script usage:
         .. code-block::
 
-            is monster_property <slot>,<property>,<value>
+            is monster_property <property>,<value>
 
     Script parameters:
-        slot: Position of the monster in the player monster list.
-        property: Property of the monster to check (e.g. "level"). Valid values
-            are:
-                - name
-                - level
-                - level_reached
```

Comment:
Why did you remove level_reached?

Reply:
I thought it was too similar to level, but I have added it again.